src/vs/workbench/common/editor.ts

```diff
@@ -154,6 +154,23 @@ export interface ITextDiffEditorPane extends IEditorPane {
  */
 export interface IEditorControl extends ICompositeControl { }
 
+export interface ICompositeCodeEditor {
+	readonly onDidChangeActiveEditor: Event<ICompositeCodeEditor>;
+	readonly activeCodeEditor: IEditor | undefined;
```

Comment:
@jrieken I wonder if eventually we should move this where `IDiffEditor` is declared and let `IDiffEditor extend ICompositeCodeEditor` because I would see use cases of knowing what the active editor is for the diff case. For example, updating the editor status depending on which side is active.

Reply:
✔️ The `ICompositeCodeEditor` interface is now is editorCommon.ts (no adoption yet for the diff editor tho). The current usages of `ICompositeCodeEditor` are the notebook editor (implements) and the `codeEditorService#getActiveCodeEditor` and `editorService#activeTextEditorControl`